overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -42854,7 +42854,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "§eInfusion?",
-          "desc:8": "In order to get higher tiers of Essence, you will need to use Infusion Crystals. They all have have unlimited durability but each crystal can only do the listed tier of essence.\n\nNote: \n\n\nEventually, you will unlock the Master Infusion Crystal, which also has unlimited durabiility, is capable of crafting every tier of essence, and can even craft blocks!\n\n\n\n... eventually.\n\n\n"
+          "desc:8": "In order to get higher tiers of Essence, you will need to use Infusion Crystals. They all have unlimited durability but each crystal can only do the listed tier of essence.\n\nNote: \n\n\nEventually, you will unlock the Master Infusion Crystal capable of crafting every tier of essence and can even craft blocks!\n\n\n\n... eventually.\n\n\n"
```

Comment:
`Eventually, you will unlock the Master Infusion Crystal capable of crafting every tier of essence and can even craft blocks!`
run on sentence, add punctuation